src/monopoly.js

```diff
@@ -1,48 +1,158 @@
-export class Monopoly {
+import BoardState from "./boardState";
+import Player from "./player";
+import fs from 'fs';
+import events from 'events';
+
+export default class Monopoly {
   constructor(){
-    this.PropertyList = null;
+    this.PropertyArray = null;
     this.ChanceList = null;
     this.CommChestList = null;
-    this.PlayerList = null;
+    this.PlayerArray = new Array(0);
+    this.currentPlayer = null;
+    this.Emitter = new events();
   }
 
-  loadPlayer(){}
+  loadPlayer(name, regID, socketID){
+    const regArray = JSON.parse(fs.readFileSync('./dist/regIDArray.json'));
+    //TODO: Check for registered ID
+    if(regArray.indexOf(regID) >= 0){
+      this.PlayerArray.push(new Player(name, regID, socketID));
+      console.log('Player ' + this.PlayerArray[this.PlayerArray.length - 1].person + ' added.');
+      if(regArray.length == this.PlayerArray.length){
+        console.log('runGame');
+        this.runGame();
+        return this;
+      }
+      return this;
+    }
+    //TODO if all ID's are registered, runGame;
+    return this;
+  }
 
   runGame(){
-    cleanBoard();
+    this.cleanBoard();
 
-    findFirstPlayer();
+    this.findFirstPlayer();
 
-    while(PlayerList.length > 1){
-      runTurn(PlayerList.current);
-      PlayerList.Current = PlayerList.Current.Next;//or next index or w.e. I use
+    while(this.PlayerArray.length > 1){
+      if(this.PlayerArray[this.currentPlayer].money < 0){//test
+        this.PlayerArray.splice(this.currentPlayer, 1);
+        continue;
+      }
+      this.runTurn(this.PlayerArray[this.currentPlayer]);
+      this.currentPlayer++;//or next index or w.e. I use
+      if(this.PlayerArray.length >= this.currentPlayer){
+        this.currentPlayer = 0;
+      }
     }
 
-    //add wincount;
+    //TODO add wincount;
   }
 
-  runTurn(){
-    
+  runTurn(ActivePlayer){
+    const thisGame = this;//because 'this' breaks with the event
+    console.log('runTurn for ' + ActivePlayer.person + '. Player position: ' + ActivePlayer.position);//test
+
+    this.Emitter.once('rollDice', function(){
+      thisGame.toggleListenersOff();
+
+      let diceArray = thisGame.rollDice();
+      console.log('dice: ' + diceArray.reduce(( acc, cur ) => acc + cur, 0));//test
+      ActivePlayer.movePlayer(diceArray, thisGame.PropertyArray.length);
+      console.log('new position ' + ActivePlayer.position);//test
+      ActivePlayer.money -= diceArray.reduce(( acc, cur ) => acc + cur, 0) * 10;//test
+//      thisGame.landOnFunction(ActivePlayer);TODO
+
+      if(diceArray.every(element => element == diceArray[0])){//probably too much.  allows for more than 2 dice
+        if(++ActivePlayer.doubles >= 3){
```

Comment:
It took me like 3 times to read through this to understand what was going on.  I generally avoid the prefix incrementer/decrementer because it's hard to visually inspect and know immediately what's going on.  

Suggest changing to:
```
activePlayer.doubles += 1;
if (activePlayer.doubles >= 3) {
// stuff
}
```